Anlab.Mvc/Views/Lab/UpdateFromCompletedTests.cshtml

```diff
@@ -46,7 +52,18 @@
             $("#table").dataTable();
         });
     </script>
-                                                                  
-    @{         await Html.RenderPartialAsync("_ShowdownScriptsPartial"); }
 
+    <script type="text/javascript">
+        $('#adjustmentAmount').on('keyup', function () {
+            var prevTotal = $('#prevTotal').val();
+            var adjustAmt = $(this).val();
+            if (adjustAmt)
+                var totalCost = parseFloat(prevTotal) + parseFloat(adjustAmt);
+            else
+                var totalCost = prevTotal;
+            $('.newTotal').html(totalCost);
```

Comment:
@laholstege update that span to have an id="newTotal" or similar and use the ID here, just to make sure you are updating only the one proper element